test/test-trace-api.js

```diff
@@ -94,13 +101,14 @@ describe('Trace Interface', function() {
     
     before(function() {
       traceAPI.enable_(agent);
-      common.init(traceAPI);
-      common.avoidTraceWriterAuth(traceAPI);
+      traceAPI.private_().traceWriter.request = request;
+      common.avoidTraceWriterAuth();
     });
 
     afterEach(function() {
-      common.cleanTraces(traceAPI);
-      common.clearNamespace(traceAPI);
+      TraceWriter.get().buffer_ = []; // clear traces
```

Comment:
Ok, what do you think of making `clearNamespace` be the only operation in common that still accepts a `traceAPI`? I would be fine with this or the existing approach.